settings.gradle.kts

```diff
@@ -0,0 +1,14 @@
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
```

Comment:
issue: You are using `FAIL_ON_PROJECT_REPOS` mode and defining repositories in `build.gradle.kts` file. You should move those repositories here